.changeset/curvy-planets-hear.md

```diff
@@ -0,0 +1,7 @@
+---
+'@theguild/components': minor
```

Comment:
let's do a major, because logo are used in https://the-guild.dev/#ecosystem
```suggestion
'@theguild/components': major
```

Reply:
Fair point.

Shall we wait with the merge then until we have all the new logos, and then update The Guild landing page as well?

Reply:
I created https://github.com/the-guild-org/docs/pull/1587 for v7 alpha releases to not block v6